src/widgets/EmbedWidget.ts

```diff
@@ -72,6 +71,9 @@ export default class EmbedWidget extends Widget {
         }
       });
 
+      // observe height right away
+      ro.observe(frameDoc.body)
+      // observe any changes afterwards
       ro.observe(await this._findInnerContainer());
```

Comment:
If the resize observer watches both, and they are different, then this will lead to jank and potentially an infinite loop. (resizes to size of body, then inner, then body, then inner, then body, then inner)

The resize observer should be set up to observe only 1 element at a time.

```suggestion
      // observe height right away
      ro.observe(frameDoc.body)
      const inner = await this._findInnerContainer();
      ro.unobserve(frameDoc.body);
      // observe any changes afterwards
      ro.observe(inner);
```